src/main/resources/graphql/team.graphqls

```diff
@@ -0,0 +1,55 @@
+
+extend type Query {
+    allTeams: [Team!]!
+    teamById(id: ID): Team
+
+
```

Comment:
Make sure to remove the whitespace and keep spacing consistent across all files.